mobile_app.md

```diff
@@ -0,0 +1,20 @@
+# MiniAirQualityStation - Mobile App
+
+# Introduction
+
+The goal of the project is to create a mobile app that visualises information provided by sensors mounted on the Raspberry Pi based mini air quality station and configure it.
+
+# Specification
+
+## Mobile OS
+
+Google Android 8.0 (Oreo) or higher
+
+## Coding language and other technologies used
+
+Java, Android WebView
+
+## Development environment
+
+Android Studio Dolphin 2022.1.1 with Gradle building system.
```

Comment:
I use Electric Eel version, but it should be compatible.

Reply:
so maybe we can simplify that to `Android Studio with Gradle building system`
?